### ISSUE TITLE
New requirement to have 'run' in command is non-obvious

### ISSUE THREAD

**Issue by zofrex** [mbj/mutant#1092]
I upgraded Mutant (from 0.9.11 -> 0.10.5) and my existing task to run it failed:

```
% bundle exec mutant --include lib --require 'runeterra_cards' --use minitest -- 'RuneterraCards*'
mutant: invalid option: --include
```

I read through the changelog and didn't see anything obvious between these two versions. After digging a bit more I realised this is due to #1073. The right command to run is now:

```
bundle exec mutant run --include lib --require 'runeterra_cards' --use minitest -- 'RuneterraCards*'
```

Would it be possible to make running `mutant` do the same as it did before, e.g. making "run" the default command, similarly to how "install" is the default command for Bundler?

If not, I think it would be nice to have a more intuitive error message here, e.g. "No command given, commands available: run, license". Additionally, the changelog could make it more obvious that a breaking change was made and what the fix is.

**Comment by mbj** [comment]
@zaidan Thanks for the report. I tested the `bundle exec mutant` case (and it has nice UI), but forgot to test with extra arguments (I rarely use these as I use the config file!).

Fixing this is easy, I just have to render the `--help` of the parent command when an invalid option is specified. Good catch.

**Comment by mbj** [comment]
> Additionally, the changelog could make it more obvious that a breaking change was made and what the fix is.

Okay, I want to improve the changelog format significantly. I'll retrofit this. Note that I intentionaly bumped from `0.9` to `0.10` to "signal" the breaking change (note we are pre 1.x, so I use shifted semver).

**Comment by mbj** [comment]
@zofrex I've released 0.10.6 with better behavior. Feel free to give me your opinion if the behavior would have helped you to navigate this situation better.

**Comment by mbj** [comment]
@zofrex This is what mutant would print for your case:

```
bundle exec mutant --include lib --require 'runeterra_cards' --use minitest -- 'RuneterraCards*'

mutant: invalid option: --include

usage: mutant <run|subscription> [options]

Summary: mutation testing engine main command

Global Options:

        --help                       Print help
        --version                    Print mutants version

Available subcommands:

run          - Run code analysis
subscription - Subscription subcommands
```

**Comment by zofrex** [comment]
@mbj yes, that would definitely have helped, I like it!

> Note that I intentionaly bumped from 0.9 to 0.10 to "signal" the breaking change

I did notice that btw, and it did make it easier to find the breaking change (and also to quickly pin a working version in CI). Much appreciated!

**Comment by mbj** [comment]
@zofrex Okay, this is released as 0.10.6. Thanks for reporting! BTW I also added the expanded display on specifying an unknown subcommand.
